pdoc/html_helpers.py

```diff
@@ -98,6 +98,11 @@ def _fenced_code_blocks_hidden(text):
     def hide(text):
         def replace(match):
             orig = match.group()
+
+            # Remove a trailing blank line before the closing ```, which might
+            # have been added while processing `.. include::` directives.
```

Comment:
```suggestion
            # Remove a trailing blank line before the closing ```, which was
            # added if the code block came in via `.. include::` directive.
```